version.php

```diff
@@ -28,7 +28,7 @@
 // is a database change in the course of development.  It is used
 // internally to determine when a database upgrade is needed.
 //
-$v_database = 498;
+$v_database = 500;
```

Comment:
wouldn't increment this for patches (ie. in the release branches) since i think it would cause weird behavior in the admin.php script (i think it would make a link to run the upgrade script instead of the patch script)